### DISCUSSION TITLE
a set up problem

### DISCUSSION THREAD

**Discussion by FuHanq** [Security-Onion-Solutions/securityonion#7323] [Unsupported Versions] [answered]
follow the set up document, when finish import installation, enter to reboot, found this erro.
![image-20220225140614575](https://user-images.githubusercontent.com/89372066/155665341-87e794d2-177c-4e2e-9f48-2e486c717a0a.png)

**Comment by dougburks** [comment, marked as answer]
`Could not resolve host` indicates a DNS issue. Double-check your network settings and your Internet connection.
